package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "purescript-lens",
-  "version": "1.0.1",
```

Comment:
Please do not remove this, it is required: https://docs.npmjs.com/files/package.json#version